### PR TITLE
qa/tests: Added /home/teuthology/rhel_only_on_ovh.yaml arg for …

### PR DESCRIPTION
**Pull request by yuriw**
…ceph-disk run on mimic

Fixes: https://tracker.ceph.com/issues/25005
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -154,7 +154,8 @@ CEPH_QA_EMAIL="ceph-qa@ceph.com"
 #30 05 * * 1,3,5 CEPH_BRANCH=mimic; MACHINE_NAME=smithi;/home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s hadoop -e $CEPH_QA_EMAIL
 #35 05 * * 1,3,5 CEPH_BRANCH=mimic; MACHINE_NAME=smithi;/home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s samba -e $CEPH_QA_EMAIL
 55 05 * * 1,3,5  CEPH_BRANCH=mimic; MACHINE_NAME=mira; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-deploy -k distro -e $CEPH_QA_EMAIL
-10 05 * * 1,3,5  CEPH_BRANCH=mimic; MACHINE_NAME=ovh;  /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s ceph-disk -k distro -e $CEPH_QA_EMAIL
+#when we run cpeh-dosk on ovh we have to use overrides from /home/teuthology/rhel_only_on_ovh.yaml
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: fixed

**Comment by smithfarm** [comment]
Given that the ceph-disk suite is known to run only on virtual machines, and the only ones we have now are in OVH, wouldn't it make sense to add

```
overrides:
  ansible.cephlab:
    skip_tags: entitlements,packages,repos
```

directly to the ceph-disk suite yaml?

**Comment by smithfarm** [comment]
@yuriw Great that you found a fix for this!

**Comment by yuriw** [comment]
@smithfarm trying this https://github.com/ceph/ceph/pull/23156

We have `ceph-disk` on mimic only, my guess is that it won't be supported in `mimic +`
We probably don't need it in `luminous` because we had no rhel support there iirc.

**Comment by yuriw** [comment]
@smithfarm Let's close this and merge https://github.com/ceph/ceph/pull/23156
what do you think ?

**Pull request closed without merging**